app/services/browse_list.rb

```diff
@@ -55,4 +55,12 @@ def facet_query
              })
         .dig('facet_counts', 'facet_fields', field)
     end
+
+    def normalize_prefix(prefix)
+      return if prefix.blank?
+
+      normalized_prefix = prefix.dup
```

Comment:
@awead - I had to add `.dup` here because otherwise the string was copied by reference and not by value, so modifying `normalized_prefix` still caused the `FrozenError`. Adding `.dup` copies the string by value and avoids overwriting the original prefix string in memory.

Reply:
Yeah, that looks familiar.